src/app/Component/ArchivedMessages/ArchivedMessages.js

```diff
@@ -0,0 +1,165 @@
+import React, { Component } from "react";
+import "./../ChatScreen/chatscreen.css";
+import Header from "../Common/Header";
+import axios from "axios";
+import { connect } from "react-redux";
+import { createClient } from "../../actions/actions";
+import { loaderService } from "../../../service/loaderService";
+
+class ChatScreen extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            Data: null,
+            user: this.props.location.state && this.props.location.state.user,
+            menu: false,
+            settingDetails: false,
+            isEmpty: false,
+        };
+        console.log(this.props);
```

Comment:
If it is needed keep console log or else remove because it will leads to performance issue .Check in other places too